source/MaterialXRuntime/RtToken.h

```diff
@@ -157,12 +157,6 @@ class RtToken
         return _entry->_str.c_str();
     }
 
-    /// Conversion to std::string.
-    operator const string& () const
```

Comment:
Seems this implicit conversion from RtToken to string is causing issues, so not worth the convenience of having it. Better to be explicit when conversions are done.